gnomad_qc/v4/resources/sample_qc.py

```diff
@@ -846,3 +846,13 @@ def finalized_outlier_filtering(test: bool = False) -> VersionedTableResource:
         for version in VERSIONS
     },
 )
+
+
+def get_sample_qc_field_def_json_path(version: str) -> str:
```

Comment:
Sorry one more thing 
```suggestion
def get_sample_qc_field_def_json_path(version: str = CURRENT_VERSION) -> str:
```